test/fixture/generatedTypeScript/interfaces/ICodeableConcept.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/prefer-default-export, import/no-cycle, @typescript-eslint/naming-convention, @typescript-eslint/no-empty-interface */
+import { 
+  ICoding,
+  IElement,
+  IPrimitiveString,
+} from "../internal";
+
+export interface ICodeableConcept extends IElement {
```

Comment:
Codacy found an issue: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)